packages/suite/src/hooks/wallet/form/useCompose.ts

```diff
@@ -88,7 +89,7 @@ export const useCompose = <TFieldValues extends FormState>({
                         formState,
                         composeContext: state,
                     }),
-                ).unwrap();
+                ).then(res => (isFulfilled(res) ? res.payload : undefined));
```

Comment:
I prefer `await` syntax:
```suggestion
               const response = await dispatch(
                    composeSendFormTransactionFeeLevelsThunk({
                        formState,
                        composeContext: state,
                    }),
                )
                
              return isFulfilled(response) ? response.payload : undefined;               
```

But it might be just my personal preference. Up to you.